loans, added reset to util, added loans_test

achyuta116 opened the pull request:
Implemented functions in `loans.py`
Added reset functionality to util.py sending users back to login screen.
Wrote tests in `tests/loans_test.py` using unittest to validate correctness of the functions.

nkilm reviewed (changes requested):
Make the following changes
- Use inquirer to take inputs from the user, it'll help to validate the inputs.
- Import `login` from `login.py` as `__main__.py` will be removed.

achyuta116 commented:
Modified `loan_tests.py` to work with the new login function implementation within `login.py`
Fixed imports in `utils/util.py`
Added input validation for within `customer_loan_application` in `loans.py`
